pkg/resource/feature_group/testdata/test_suite.yaml

```diff
@@ -0,0 +1,17 @@
+tests:
+  - name: "Feature group demo test"
+    description: "Feature group CRD tests"
+    scenarios:
+     - name: "Create=InvalidInput"
+       description: "Given one of the parameters is invalid, ko.Status shows a terminal condition"
+       given:
+         desired_state: "feature_group/cr/fg_cmd_invalid_before_create.yaml"
+         svc_api:
+           - operation: CreateFeatureGroupWithContext
+             error:
+               code: InvalidParameterValue
+               message: "The feature group name must start with an alphanumeric character."
+       invoke: Create
+       expect:
+         latest_state: "feature_group/cr/fg_cmd_invalid_create_attempted.yaml"
+         error: resource is in terminal condition
```

Comment:
does this string have any meaning?

Reply:
Yes, this is the [standard terminal error message string](https://github.com/aws-controllers-k8s/runtime/blob/d7b6e45c7ba9e54664b49f681747abf275a232f9/pkg/errors/error.go#L53) that is set on terminal condition. It is set in the call OnError in manager.go, after the resource comes back from an updateConditions after sdkCreate with the terminal condition set.

Reply:
got it `runTestScenario` in `pkg/testutil/test_suite_runner.go`